docs/sdk/dev/development-tools/architecture-sniffer.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Architecture sniffer
-description: Use Architecture Sniffer to assert a certain quality of Spryker architecture for both core and project
+title: Architecture Sniffer
```

Comment:
What is "sniffer" uppercase?

Reply:
For sniffer and linter, there were discrepancies in the documentation - there were uses of Sniffer / Linter and uses of sniffer / linter in both titles and descriptions. To be consistent I opted for Sniffer / Linter, assuming that in this case they were specific tools and thus Sniffer was part of its official name as opposed to just descriptors. If its the opposite, then we can switch to sniffer / linter @VadymSachenko?